### ISSUE TITLE
Single Cell Datasets: reduce the size of the data sets

### ISSUE THREAD

**Issue by BlazZupan** [biolab/orange3-single-cell#208]
This issue is related to PR https://github.com/biolab/orange3/pull/3047, which enables saving and loading of compressed pickle files. Once this is merged into Orange and released, I propose to:

- move all the raw files to file.biolab.si/datasets/sc and there provide a compressed pickled versions
- update info files for Single Cell Datasets accordingly

This should substantially reduce the transfer and loading time of data sets. For instance, the largest data set currently included (bone marrow with AML) has 64MB, while its pickled xz variant has on 2.4MB.

This update will create an issue with backward compatibility, which will be broken.

**Comment by astaric** [comment]
If we will migrate to .pickle.xz, I suggest we create a new "repository on serverfiles" and migrate to that. Otherwise, Datasets will start crashing on old(er) versions of the software.

**Comment by anupparikh** [comment]
We're creating another file format to support? why not standardize on tab and loom? 

tab, mtx, loom, pickle...

**Comment by BlazZupan** [comment]
@anupparikh, it's not a new format, it is just the way of storing tab (csv) files. A quick trick to substantially reduce the size of our demo datasets and speed-up the loading.

**Comment by anupparikh** [comment]
So this is just for demo datasets, customers won't be creating and using
pickled data.
